packages/sources/react/src/components/indicators/VtmnProgressbar/VtmnProgressbar.tsx

```diff
@@ -82,21 +82,30 @@ export const VtmnProgressbar = ({
         className,
       )}
       role="progressbar"
-      aria-label="progress bar"
       aria-valuemin={0}
       aria-valuemax={100}
+      aria-valuenow={value}
       {...props}
     >
       {/**
        * Linear Progressbar
        */}
 
-      {variant === 'linear' && status === 'determinate' && (
-        <span
-          className="vtmn-progressbar_label"
-          // Clamp the value between minValue and maxValue
+      {variant == 'linear' && status == 'determinate' && (
+        <div className="vtmn-progressbar_label">
+          <span id={...props['aria-labelledby'] || undefined}>
```

Comment:
Just ...props['aria-labelledby'] should be ok :)

Reply:
If the aria-label is not set it creates an id with undefined value. I want not to create an id if no aria-labelled is passed.

Reply:
But if `props['aria-labelledby']` are not set, it should be undefined ? 🤔

Reply:
it should be undefined yes, but not to have an attribute set to "undefined". I fixed this. Thanks :)